hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/redis/RedisEntityListenerIntegrationTest.java

```
@@ -47,19 +47,19 @@ public RedisEntityListenerIntegrationTest(RedisEntityListener entityListener, Re
 
     @Override
     protected Flux<TopicMessage> subscribe(long topicNum) {
-        UnicastProcessor<TopicMessage> processor = UnicastProcessor.create();
+        Sinks.Many<TopicMessage> sink = Sinks.many().unicast().onBackpressureBuffer();
         RedisSerializer stringSerializer = ((RedisTemplate<String, ?>) redisOperations).getStringSerializer();
         RedisSerializer<TopicMessage> serializer = (RedisSerializer<TopicMessage>) redisOperations.getValueSerializer();
 
         RedisCallback<TopicMessage> redisCallback = connection -> {
             byte[] channel = stringSerializer.serialize("topic.0.0." + topicNum);
             connection.subscribe((message, pattern) -> {
-                processor.onNext(serializer.deserialize(message.getBody()));
+                sink.tryEmitNext(serializer.deserialize(message.getBody()));
```

Should we be checking the EmitResult to distinguish between the results e.g. OK, vs  cancelled/overflow/terminated/zero subscriber?

---

True, changed to emitNext with a fail fast handler. This replicates the previous behavior when using the Processors.